.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Comment:
@aacic Since the `.devcontainer/Dockerfile` only has one line to use the base dev image, should this line just be `image: ....` instead?

Reply:
I added the Docker file so someone could extend it if needed. But, let's keep it simple. I've replaced this with the image tag.